RFC-001-BAM.md

```diff
@@ -247,13 +243,12 @@ A header's key acts as its identifier. A header's value MUST encode the followin
 
 2. A value: The actual value that is associated with the header
 
-    A header's value MUST be a single token, e.g. no nested structure etc. This is to facilitate comparisons of header-values. If a value needs to parameterized further, `parameters` should be used. Every header MUST have a value.
+    Every header MUST have a value. A header's value determines, what the `parameters` look like.
 
 3. Parameters
 
-    Parameters are to be used to further specify a header's value. Parameters are a key-value structure with **unique** keys. Similar to the actual header value, both keys and values must be single tokens. Parameters are optional and may therefore be empty or left out if the encoding allows this.
-
-Headers can occur multiple times, e.g. a single header-key can map to multiple values where all of them have one `value` and `parameters`. How this is represented depends on the encoding.
+    Parameters are additional data that depend on a header's value.
+    Parameters are optional and may therefore be empty or left out if the encoding allows this.
```

Comment:
❌ Actually aren't some parameters mandatory? It's just that the possible parameters per header value must never change. (I guess you could add some non-mandatory parameters?). The network, for example might be mandatory or the `hash_function` for `comit-rfc-003`.

Is correcting this outside the scope of this PR?

Reply:
You mind suggesting a re-phrase?

`Depending on a header's value, parameters might be entirely optional and MAY therefore be empty or left out if the encoding allows this.`